lib/rummager/config.rb

```diff
@@ -3,6 +3,16 @@
 
 configure :development do
   set :protection, false
+  set :host_authorization, {
+    permitted_hosts: [
+      "search-api.dev.gov.uk",
+      "localhost",
+      ".localhost",
+      ".test",
+      IPAddr.new("0.0.0.0/0"),
+      IPAddr.new("::/0"),
```

Comment:
```suggestion
      ".dev.gov.uk",
      "localhost",
      IPAddr.new("0.0.0.0/0"),
      IPAddr.new("::/0"),      
```

I think we can shorten this a bit as it'd be very unexpected for someone to be using a .localhost or .test domain locally given we have the .dev.gov.uk convention.

Do also wonder if this is worth anything in a dev environment and we might want to set it as an empty array to disable it? but maybe I'm missing something? If I understand right the IP addresses will match basically any IPv4 or IPv6 addresses, so it's very broad.